building/windows.md

```diff
@@ -0,0 +1,231 @@
+# Windows
+
+For now, it is possible to build Phoenix-RTOS images for all targets and run `ia32-generic-qemu` on Windows using
+`Cygwin` or `WSL`. Running the rest of targets is under development.
+
+## Cygwin
+
+To start, we need to download and set the QEMU environment.
+
+To do this, go to the official [QEMU website](https://www.qemu.org/download/#windows) and select a mirror targeted
+for your system.
+
+![OS bit selection](_images/qemu-os-select.png)
+
+Next up we need to download `Cygwin` from the [official website](https://cygwin.com/install.html). It is a collection of
+Open Source tools that provide
+
+<details>
+<summary>`Cygwin` installation/update process (click to expand)</summary>
+&nbsp;
+
+* The installation process also serves as a package manager. `Cygwin` will ask about a type of installation in our case
+it will be from the internet.
+
+![cygwin-download-source](_images/cygwin-download-source.png)
+
+* After that, the installation process will ask about the placement of a root and local package directory
+
+![cygwin-root-dir](_images/cygwin-root-dir.png)
+![cygwin-local-directory](_images/cygwin-local-directory.png)
+
+* `Cygwin` has the capability of using host's internet connection, but it needs to be configured with our preferences
+(In most scenario it will be System Proxy Settings)
+
+![cygwin-internet-connection](_images/cygwin-internet-connection.png)
+
+* Selection of a mirror for package downloads can be tricky, sometimes it needs to be changed for better results because
+not all mirror have all available packets.
+
+![cygwin-mirror-selection](_images/cygwin-mirror-selection.png)
+
+* After selecting the mirror we can select the packages we want. But in larger groups, this can be annoying. That's why
+we only select two elements (They will be installed with dependencies) which are mandatory to use package manager
+`apt-cyg` in `Cygwin` terminal.
+
+![cygwin-selecting-packages](_images/cygwin-selecting-packages.png)
+
+* Once these are chosen, proceed to run the installation process.
+
+</details>
+  &nbsp;
+
+If installation process was performed, next step is to run `Cygwin` terminal.
+Always run it as administrator, the best way to do that is simply setup this as default action:
+
+![cygwin-admin-rights](_images/cygwin-admin-rights.png)
+
+  <details>
+<summary>Installing apt-cyg and mandatory packages (click to expand)</summary>
+
+  &nbsp;
+<details>
+<summary>Where place listed commands?</summary>

```

Comment:
Yes because sometimes Cygwin can map full directory path to home in example: `/cygdrive/c/cygwin64/home/user/` but home directory should be `/home/user/`